pom.xml

```diff
@@ -13,6 +13,12 @@
   <name>Jenkins jQuery plugin</name>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/jQuery+Plugin</url>
 
+  <properties>
+    <jenkins.version>1.625</jenkins.version>
+    <jenkins-test-harness.version>1.625</jenkins-test-harness.version>
```

Comment:
The JTH comes from the parent pom, in this case without overriding it would be 2.18 https://github.com/jenkinsci/plugin-pom/blob/plugin-2.25/pom.xml#L52 which is not compatible with the core version used